plugins/stats_over_http/stats_over_http.cc

```diff
@@ -726,9 +743,8 @@ stats_origin(TSCont contp, TSEvent /* event ATS_UNUSED */, void *edata)
   /* This is us -- register our intercept */
   Dbg(dbg_ctl, "Intercepting request");
 
-  my_state = (stats_state *)TSmalloc(sizeof(*my_state));
-  memset(my_state, 0, sizeof(*my_state));
-  icontp = TSContCreate(stats_dostuff, TSMutexCreate());
+  my_state = new stats_state;
```

Comment:
You switched this to use `new`, but the cleanup of this still uses `TSfree` instead of `delete` (in `stats_cleanup`)

Reply:
I found another one too while fixing this. :)